src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDiffieHellmanCng.Key.cs

```diff
@@ -108,7 +108,7 @@ private CngKey GetKey()
             }
             else
             {
-                CngAlgorithm algorithm = null;
+                CngAlgorithm? algorithm = null;
```

Comment:
NIT: this and above variable `key` seems not need to be nullable and null assignment is redundant